dapps/marketplace/test/_services.js

```diff
@@ -38,7 +38,11 @@ export default async function() {
           width: 1280,
           height: 1024
         },
-        args: ['--no-sandbox', '--disable-setuid-sandbox']
+        args: [
+          '--no-sandbox',
+          '--disable-setuid-sandbox',
+          process.env.DISABLE_GPU === 'true' ? '--disable-gpu' : ''
```

Comment:
I think we can just disable it globally - no need to put behind a flag. I don't think it actually gets used in headless mode anyway according to some GitHub comments I read

Reply:
I think you OSX folks (and none linux+intel folks) would actually be getting some benefit from this in test performance.  This was something I explicitly needed for myself because of this stupid driver issue(I have the same problem with regular Chromium and electron apps).

Reply:
Using the flag didn't have a noticeable effect for me on OSX 🤷‍♂️

Reply:
alright, doing the reverse.  if `DISABLE_GPU` is `false` it won't use the flag.  Don't come crying to me when your CPUs melt a hole in your laptop!